doc/catalog-format-spec.rst

```diff
@@ -379,3 +398,4 @@ Parameters
 * ``compiler``: name of the current compiler. Currently supported values are:
   ``gnat-unknown``, ``gnat-fsf-old``, ``gnat-fsf-7.2``, ``gnat-fsf-7.3``,
   ``gnat-gpl-old``, ``gnat-gpl-2017``, ``gnat-community-2018``.
+* ``word-size``: architecture word size. Currently supported values are: ``bits-32``, ``bits-64``, ``bits-unknown``
```

Comment:
How will that play with cross compilation?

I guess you will need the user to specify the target architecture.

Reply:
Yes. My idea is that there will be some way for the user to override the detection of the native platform, to instead provide whatever applies to the target.